### PR TITLE
BACK41 add tests for PostInternalRestController

### PR DESCRIPTION
**Pull request by n-bert**

### CODE REVIEW

File: src/test/java/kata/academy/eurekacontentservice/rest/inner/PostInternalRestControllerTestIT.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.junit.jupiter.api.Test;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.jdbc.Sql;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+class PostInternalRestControllerTestIT extends SpringSimpleContextTest {
```
Comment (kaitanovwork) [inline src/test/java/kata/academy/eurekacontentservice/rest/inner/PostInternalRestControllerTestIT.java:13]: public

File: src/test/java/kata/academy/eurekacontentservice/rest/inner/PostInternalRestControllerTestIT.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, value = "/scripts/inner/PostInternalRestController/existsByCommentId_SuccessfulTest/BeforeTest.sql")
+    @Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, value = "/scripts/inner/PostInternalRestController/existsByCommentId_SuccessfulTest/AfterTest.sql")
+    void existsByCommentId_SuccessfulTest() throws Exception {
```
Comment (kaitanovwork) [inline src/test/java/kata/academy/eurekacontentservice/rest/inner/PostInternalRestControllerTestIT.java:18]: public + existsByPostId

File: src/test/java/kata/academy/eurekacontentservice/rest/inner/PostInternalRestControllerTestIT.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        Long postId = 1L;
+        mockMvc.perform(get("/api/internal/v1/content/posts/{postId}/exists", postId)
+                .contentType(MediaType.APPLICATION_JSON))
+            .andExpect(status().isOk())
+            .andExpect(content().string("true"));
+    }
+
+    @Test
+    void existsByCommentId_FailTest() throws Exception {
```
Comment (kaitanovwork) [inline src/test/java/kata/academy/eurekacontentservice/rest/inner/PostInternalRestControllerTestIT.java:27]: тоже самое

File: src/test/java/kata/academy/eurekacontentservice/rest/inner/PostInternalRestControllerTestIT.java (hunk 1 above)
Comment (n-bert) [reply src/test/java/kata/academy/eurekacontentservice/rest/inner/PostInternalRestControllerTestIT.java:13]: В Junit 5 написано: 
> Test classes, test methods, and lifecycle methods are not required to be public, but they must not be private.
> It is generally recommended to omit the public modifier for test classes, test methods, and lifecycle methods unless there is a technical reason for doing so <...>

вот [здесь](https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods).
У нас особый случай? :)

**Review by kaitanovwork — APPROVED** [review]

**Pull request merged**